5.machine_learning/3.assess_generalizability/README.md

```diff
@@ -0,0 +1,57 @@
+# Assess generalizability of the models
+
+In this submodule, we perform various analysis to determine how to models we trained perform on holdout plates of data that has never been seen.
+We apply the models (all features, actin only, without actin/rest) to 3 different holdout plates.
```

Comment:
```suggestion
We apply the models (all features, actin only, without actin) to 3 different holdout plates.
```

It seems like the /rest is the name of the model?

Reply:
Correct! I think people will understand without having to explicitly tell them.